src/components/NetworkSelectorContainer.js

```diff
@@ -21,22 +28,19 @@ class AdminContainer extends Component {
     const { INJECTED, LEDGER_MAIN, LEDGER_ROPSTEN } = NETWORK_TYPES;
 
     return (
-      <div className="fr ba">
-        <select
-          className="pa2"
-          value={this.props.network}
-          onChange={this.handleNetworkChange}
-          style={{
-            backgroundColor: "white",
-            borderWidth: 0,
-            borderRadius: 0
-          }}
-        >
-          <option value={INJECTED}>Metamask</option>
-          <option value={LEDGER_MAIN}>Ledger Nano (Mainnet)</option>
-          <option value={LEDGER_ROPSTEN}>Ledger Nano (Ropsten)</option>
-        </select>
-      </div>
+      <React.Fragment>
+        <div className="fr ba" css={css(divBase)}>
+          <select
+            className="pa2 provider-selector"
+            value={this.props.network}
+            onChange={this.handleNetworkChange}
+          >
+            <option value={INJECTED}>Metamask</option>
+            <option value={LEDGER_MAIN}>Ledger Nano (Mainnet)</option>
+            <option value={LEDGER_ROPSTEN}>Ledger Nano (Ropsten)</option>
+          </select>
+        </div>
+      </React.Fragment>
```

Comment:
dont need <React.Fragment> for this case.

Reply:
Updated.